src/trace/context.ts

```diff
@@ -334,13 +497,31 @@ export function readTraceFromEvent(event: any): TraceContext | undefined {
     return readTraceFromHTTPEvent(event);
   }
 
+  if (isSNSEvent(event)) {
+    return readTraceFromSNSEvent(event);
+  }
+  // TODO [astuyve] SNSSQS must be before SQS
```

Comment:
You might want to move isSNSSQSEvent in this case

Reply:
Do you know if tests hit this section of code? It might be important to ensure we get it right and keep getting it right in the future.

Reply:
I'll add better tests for this to ensure it's hit.

Reply:
We should remove this `TODO` since it looks like `isSNSSQSEvent` is being hit before `isSQSEvent` and just leave the comment.